VEDA/settings/production.py

```diff
@@ -7,16 +7,18 @@
 
 DEBUG = False
 TEMPLATE_DEBUG = DEBUG
+DEFATULT_SERVICE_VARIANT_NAME = 'video-pipeline'
```

Comment:
Don't we need this in case of local environment or test environment?

Reply:
I don't think so.